packages/graphql-toolbox/src/modules/EditorView/docexplorer/TypeLink.tsx

```diff
@@ -6,6 +6,7 @@
  */
 
 import { GraphQLList, GraphQLNonNull, GraphQLType, GraphQLNamedType } from "graphql";
+import { Fragment } from "react";
```

Comment:
This file/component is from an external source, almost exclusively prettier updates.